packages/charts/src/charts/metric/metric-chart.component.tsx

```diff
@@ -187,9 +179,11 @@ export const MetricChart: FC<Props> = ({
         )}
       </ValueContainer>
       {icon.enabled && (
-        <div style={createMargins(icon.margins)}>
+        <IconContainer
+          position={icon.position}
+          margins={createMargins(icon.margins)}
```

Comment:
How is this suppose to work - as `<IconContainer />` do not support margins property correctly ?